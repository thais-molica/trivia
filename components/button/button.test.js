import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Button from "./index";

it("Button renders correctly", () => {
  const tree = renderer.create(<Button label="btn" onClick={() => console.log('click')} />).toJSON();
  expect(tree).toMatchSnapshot();
});
