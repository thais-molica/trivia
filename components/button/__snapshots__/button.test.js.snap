// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="jsx-1173346177"
  onClick={[Function]}
>
  btn
</button>
`;
